### PR TITLE
Support for new windows kext

### PR DESCRIPTION
**Pull request by vlabo**

### CODE REVIEW

File: windows_kext/release/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "release"
+version = "0.1.0"
```
Comment (dhaavi) [inline windows_kext/release/Cargo.toml:3]: Should this version match version.txt?

File: windows_kext/release/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+3. Sing the cab file
+
+### Let Microsoft Sign
+- Go to https://partner.microsoft.com/en-us/dashboard/hardware/driver/New
+- Enter "PortmasterKext vX.X.X #1" as the product name
+- Upload `portmaster-kext_vX-X-X.cab`
+- Select the Windows 10 versions that you compiled and tested on
+- Wait for the process to finish, download the `.zip`.
+
+The zip will contain the release files.
```
Comment (dhaavi) [inline windows_kext/release/README.md:26]: Please remove the zip file from the repo.
We then should probably squash the PR to remove it from the git history.

File: windows_kext/wdk/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wdk"
+version = "0.1.0"
```
Comment (dhaavi) [inline windows_kext/wdk/Cargo.toml:3]: Again, version?

**Review by dhaavi — CHANGES_REQUESTED** [review]
Just a couple notices on first read for now.

File: windows_kext/release/Cargo.toml (hunk 1 above)
Comment (vlabo) [reply windows_kext/release/Cargo.toml:3]: This is used only by the rust build system.
It's only necessary if you want to use it with the package manager.
The libraries in the repository are refer by relative path, so the version here does not matter.

File: service/firewall/interception/windowskext2/handler.go [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+					payload:        conn.Payload,
+					verdictSet:     abool.NewBool(false),
+				}
+				new.Base.Payload()
+				info := new.Info()
+				info.Inbound = conn.Direction > 0
+				info.InTunnel = false
+				info.Protocol = packet.IPProtocol(conn.Protocol)
+				info.PID = int(conn.ProcessId)
+				info.SeenAt = time.Now()
```
[the remark this answers is not in the document]
Comment (vlabo) [reply service/firewall/interception/windowskext2/handler.go:57]: No.
Is there a benefit if its from the kext?

[the remark this answers is not in the document]
Comment (dhaavi) [reply service/firewall/interception/windowskext2/handler.go:56]: Not a big one, just wanted to know.

**Pull request merged**